Notifier/Model/ChannelRepository.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Copyright © MageSpecialist - Skeeller srl. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace MSP\Notifier\Model;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use MSP\Notifier\Model\Channel\Command\DeleteInterface;
+use MSP\Notifier\Model\Channel\Command\GetByCodeInterface;
+use MSP\Notifier\Model\Channel\Command\GetInterface;
+use MSP\Notifier\Model\Channel\Command\GetListInterface;
+use MSP\Notifier\Model\Channel\Command\SaveInterface;
+use MSP\NotifierApi\Api\ChannelRepositoryInterface;
+use MSP\NotifierApi\Api\ChannelSearchResultsInterface;
+use MSP\NotifierApi\Api\Data\ChannelInterface;
+
+/**
+ * @SuppressWarnings(PHPMD.ShortVariable)
```

Comment:
It is not needed in this class, I probably forgot when I first created an override of `setId` since it was complaining about the short variable name `$id`.